src/main/java/betteradvancements/gui/GuiBetterAdvancement.java

```diff
@@ -110,15 +113,30 @@ public void drawConnectivity(int scrollX, int scrollY, boolean drawInside) {
             int borderLineColor = 0xFF000000;
             
             if (this.betterDisplayInfo.drawDirectLines()) {
-                double x1 = scrollX + this.x + ADVANCEMENT_SIZE / 2;
+                double x1 = scrollX + this.x + ADVANCEMENT_SIZE / 2 + 3;
```

Comment:
This `+3` is the magic center value?

Reply:
Indeed it is. I dont know how, or why, but it works.